google-home-community.groovy

```diff
@@ -1507,15 +1507,15 @@ private deviceTraitPreferences_SensorState(deviceTrait) {
                         name: "${deviceTrait.name}.sensorTypes.${sensorType.key}.availableStates",
                         title: "Google Home Available States for ${sensorType.key}",
                         type: "enum",
-                        multiple: "true",
-                        required: "false",
+                        multiple: true,
+                        required: true,
```

Comment:
These all changed from `required: "false"` to `required: true`.  Are they supposed to be required or not?  I'm guessing they were required before since the string `"false"` is truthy?

Reply:
Yes, they are required.